### ISSUE TITLE
Syntax highlighting for module variables

### ISSUE THREAD

**Issue by cdce8p** [microsoft/pylance-release#964]
<h3> Environment data </h3>

-   Language Server version: 2021.2.3
-   OS and version: darwin x64
-   Python: 3.9.1
-   python.analysis.typeCheckingMode: off

<h3> Expected behaviour </h3>

Module variables are highlighted as variables.

<h3> Actual behaviour </h3>

No highlighting is applied.

<h3> Logs </h3>

--

<h3> Code Snippet / Additional information </h3>

```python
__name__
__file__
__doc__
__package__
```

<img width="872" alt="Screen Shot 2021-02-18 at 19 42 46" src="https://user-images.githubusercontent.com/30130371/108405550-7d809c80-7221-11eb-9756-f71ffba6af23.png">

**Comment by bschnurr** [comment]
this._addBuiltInSymbolToCurrentScope('__package__', node, 'str');

semantic"TokenProvider.ts is marking this as token type 'intrinsic' ie. builtin

 case DeclarationType.Intrinsic:



Not sure what the right fix is here. new colour for intrinsics with dunder in the name?


```
          this._addBuiltInSymbolToCurrentScope('__doc__', node, 'str | None');
                this._addBuiltInSymbolToCurrentScope('__name__', node, 'str');
                this._addBuiltInSymbolToCurrentScope('__loader__', node, 'Any');
                this._addBuiltInSymbolToCurrentScope('__package__', node, 'str');
                this._addBuiltInSymbolToCurrentScope('__spec__', node, 'Any');
                this._addBuiltInSymbolToCurrentScope('__path__', node, 'Iterable[str]');
                this._addBuiltInSymbolToCurrentScope('__file__', node, 'str');
                this._addBuiltInSymbolToCurrentScope('__cached__', node, 'str');
                this._addBuiltInSymbolToCurrentScope('__dict__', node, 'Dict[str, Any]');
                this._addBuiltInSymbolToCurrentScope('__annotations__', node, 'Dict[str, Any]');
                this._addBuiltInSymbolToCurrentScope('__builtins__', node, 'Any');
```

other intrinsics
```
export type IntrinsicType = 'Any' | 'str' | 'str | None' | 'int' | 'Iterable[str]' | 'class' | 'Dict[str, Any]';
```

**Comment by bschnurr** [comment]
Looks like we added a tokenModifier for builtins
https://github.com/microsoft/pyrx/pull/692
`support.function.builtin.python`

I'm going to make intrinics with dunder in the name a variable type with builtin modifier
`support.variable.builtin.python`

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.9.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202191-8-september-2021
